pyproject.toml

```diff
@@ -1,6 +1,10 @@
 # Specifying the build system - PEP 517
 
 [build-system]
-requires = ["setuptools", "wheel", "numpy"]
+requires = [
+    "setuptools", "wheel",
+    "numpy==1.16.0; python_version=='3.7' and platform_machine!='aarch64'",
+    "numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64'",
+    "numpy==1.19.3; python_version=='3.9' and platform_machine!='aarch64'",
+    "numpy==1.19.3; platform_machine=='aarch64'"]
```

Comment:
I had to do this instead of `oldest-supported-numpy` because we need `numpy >= 1.16.0` also for Python 3.7 for a bug fixed in f2py that prevented build.

Reply:
Sollten wir ein numpy requirement für python 3.10 angeben? Beziehungsweise `3.9` → `>=3.9`

Reply:
Done (und auch `<=3.7`)